### DISCUSSION TITLE
dynamically adding jobs

### DISCUSSION THREAD

**Discussion by va-fivenine** [agronholm/apscheduler#936] [Q&A] [answered]
I want to be able to read a configuration file from disk that I can construct a schedule ...initially want to be able to add_job at minimum would like to pass parameters for "cron" into the add_Job as a string.  Is this possible?

i.e.
cron_schedule = "hour= 17, minute= '*'"
scheduler = BlockingScheduler()
job_id = scheduler.add_job(display, 'cron', f'{cron_schedule}' ,args=["Job"])   
scheduler.start()

**Comment by agronholm** [comment, marked as answer]
I think your best bet is to create a [crontab](https://www.adminschoice.com/crontab-quick-reference)-style file and feed it to `CronTrigger.from_crontab(...)`, line by line.
